babel.config.js

```diff
@@ -46,17 +58,8 @@ module.exports = {
   ],
   ignore: [/@babel[\\|/]runtime/],
   env: {
-    test: {
-      sourceMaps: 'both',
-      plugins: [
-        [
-          'babel-plugin-module-resolver',
-          {
-            root: ['./'],
-            alias: defaultAlias,
-          },
-        ],
-      ],
+    cjs: {
+      plugins: [...productionPlugins],
```

Comment:
Do we need to clone the array?

Reply:
Not really. I guess this is just left from reducing the plugins to a single group. Will fix it later.